locales/hin.json

```diff
@@ -443,7 +443,29 @@
       },
       "rejected": {
         "title": "सूचना",
-        "message": "आपका कार्ड {{रिसीवर}} द्वारा खारिज कर दिया गया था"
+        "message": "आपका कार्ड ने अस्वीकार कर दिया था {{receiver}}"
+      }
+    },
+    "WelcomeScreen": {
```

Comment:
did we add these extra fields intentionally

Reply:
No I got this during merge conflict resolution and since the templates have to be in English I chose the `English, {{receiver}}` one instead of the `Hindi {{receiver}}` one.